lang/janet/Portfile

```
@@ -2,11 +2,15 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           legacysupport 1.1
 PortGroup           makefile 1.0
 
+# strnlen, O_CLOEXEC
+legacysupport.newest_darwin_requires_legacy 10
+
 github.setup        janet-lang janet 1.39.0 v
 github.tarball_from archive
-revision            0
+revision            1
```

It doesn't look like the rest of the changes change anything for users on macOS 10.15 and later, and it didn't build before on macOS 10.14 and later, therefore no revision increase is needed. See https://guide.macports.org/#reference.keywords
```suggestion
revision            0
```